### ISSUE TITLE
Gradient overflow.  Skipping step, loss scaler 0 reducing loss scale to 16384.0

### ISSUE THREAD

**Issue by licaizi** [LUSSeg/PASS#9]
Hi, thanks for your wonderful work. I have conducted the pretraining experiment main_pretrain.py in luss50_pass.sh, it seems like the training process is finished normally except the gradient overflow, I'm not sure if it is ok for this experiment, I have only change a few settings in luss50_pass.sh (my conda environment: pytorch 2.1.2 + cuda 12.1 + apex 0.1):
=================================================================
[CUDA='0,1,2,3'
N_GPU=4
BATCH=64]
...
[CUDA_VISIBLE_DEVICES=${CUDA} torchrun --standalone --nproc_per_node=${N_GPU} main_pretrain.py \]
==================================================================

![image](https://github.com/LUSSeg/PASS/assets/6143725/42ec02c8-7fee-4f74-b4e8-54984f34f538)

**Comment by gasvn** [comment]
It's ok to show this gradient overflow warning. The code will automaticly adjust this.

**Comment by licaizi** [comment]
@gasvn Thanks very much for your reply and your excellent work!
